mysql-test/suite/innodb/include/percona_temp_table_encrypt.inc

```diff
@@ -6,7 +6,7 @@ call mtr.add_suppression("\\[Error\\] InnoDB: Encryption can't find master key,
 
```

Comment:
You also have to adjust plugin/keyring_vault/tests/mtr/temp_table_encrypt_keyring_vault